scripts/curio.service

```diff
@@ -1,12 +1,16 @@
 [Unit]
 Description=Curio
 After=network.target
-After=lotus-daemon.service
+# After=lotus-daemon.service  # Should not be on the same machine.
```

Comment:
We don't care if the daemon is running or not as a service. We should not depend on any service except the OS ones. Probably should remove that line altogether.